### ISSUE TITLE
Including files outside of the current directory hierarchy via package.include

### ISSUE THREAD

**Issue by danielkoo** [serverless/serverless#1896]
Hi,

package.include seems to only allow the packaging of files in the current directory or below as, if I try to specify something like

`../handler.js`

this file is not packaged up in the ZIP file, which makes some sense as otherwise this would clobber an identically-named file in the current dir.

However, it seems we can refer to resource files outside of the current hierarchy:
https://github.com/danielkoo/serverless/blob/master/docs/understanding-serverless/serverless-yml.md
```
resources:
  Resources:
    $ref: ../custom_resources.json # you can use JSON-REF to ref other JSON files
```

##### Benefits:

* Benefit one: it allows for more flexibility in setting up the dir structure, instead of requiring every file to sit under the one dir.

##### Drawbacks:

* the resultant structure can become too complicated, and may result in breakages if people move files around without being aware of the references in serverless.yml.
* there is a risk a file outside of a project's repo is included, which would break when this is cloned into another environment that does not have the same file
* the bahaviour of 'include' would be different to 'exclude', as exclude works on the current dir and below, as there's no point excluding the parent dir and above :)

##### TLDR:
I guess I'd like to start a discussion on the pros and cons of having one directive/setting doing it one way, and another doing it a different way.

**Comment by pmuens** [comment]
Hey @danielkoo thanks for this issue.

Here's another one which is related and shows why we went with the "zip per service" approach and how you can resolve this issue with the help of reusable npm packages: https://github.com/serverless/serverless/issues/1819

**Comment by flomotlik** [comment]
Thanks for reporting @danielkoo . As @pmuens already mentioned there are other discussions about this already and we decided that for now we're not going to support this feature due to potential complications it can lead to and also potential security issues with it. Therefore closing but its linked to other issues now if we ever want to come back to it.

**Comment by danielkoo** [comment]
Thanks for the comments and the links to the related issues guys, it's clearer to me now what the design decisions were, which make sense. I just need to fix the tooling/package management on my end to make it work :)
